### PR TITLE
Fix Swap function for lfu.priorityQueue.

### PR DESCRIPTION
**Pull request by chenyanchen**
Make the `entry`'s index right.

### CODE REVIEW

**Comment by chenyanchen** [comment]
Fix #39

**Comment by chenyanchen** [comment]
I found a workflow in `.github` directory, why `Test` action not checked? Did you add a Branch protection for `main`.

File: policy/lfu/priority_queue.go [hunk 1]
```diff
@@ -40,29 +40,29 @@ func newPriorityQueue[K comparable, V any](cap int) *priorityQueue[K, V] {
...
 }
 
-func (l priorityQueue[K, V]) Swap(i, j int) {
-	l[i], l[j] = l[j], l[i]
-	l[i].index = i
-	l[i].index = j
+func (q priorityQueue[K, V]) Swap(i, j int) {
+	q[i], q[j] = q[j], q[i]
+	q[i].index = i
+	q[j].index = j
```
Comment (Code-Hex) [inline policy/lfu/priority_queue.go:55]: @chenyanchen Did you fix just this line?

Comment (chenyanchen) [reply policy/lfu/priority_queue.go:55]: Yes

**Comment by Code-Hex** [comment]
I approved to run actions in this PR for now

**Comment by chenyanchen** [comment]
What's wrong with `Test(^1.19)`

**Review by Code-Hex — APPROVED** [review]
LGTM!
Thank you

**Pull request merged**